src/app/visual/repeater/repeater-visual.component.html

```diff
@@ -73,7 +73,23 @@ <h3>Collapsed repeater</h3>
       <sky-repeater-item-title>Title 2</sky-repeater-item-title>
       <sky-repeater-item-content>Content 2</sky-repeater-item-content>
     </sky-repeater-item>
+    <sky-repeater-item
```

Comment:
I avoided that because it would have made this PR bigger (there's a lot in this file), and I didn't want to be mean to the reviewer. Since you asked though, I'll do it :-P

Reply:
Done.